src/pages/City/components/NearestCitiesWeatherWidget/NearestCitiesWeatherWidget.tsx

```diff
@@ -0,0 +1,28 @@
+import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
+import { CityWeatherShortcutLoader } from 'components/CityWeatherShortcut/components/CityWeatherShortcutLoader';
+import { Place } from 'models/Place';
+import { Container } from './NearestCitiesWeatherWidget.styled';
+import { NearestCitiesWeatherWidgetTestIds } from './NearestCitiesWeatherWidgetTestIds';
+
+interface Props {
+  loading: boolean;
+  cityName: string | undefined;
+}
+
+const NearestCitiesWeatherWidget = ({ loading, cityName }: Props) => {
```

Comment:
**question** why do you need a loading flag here?

Reply:
Because at first, when we are waiting for the response from nearest cities api, we don't even know the amount of cities, so I use it as shown in line 14. When we get the data and know amount the of returned cities, we pass data and then CityWeatherShortcut shows loader until it gets data about weather